src/React/DOM/Props.js

```diff
@@ -1,8 +1,5 @@
-/* global exports */
 "use strict";
 
-var React = require("react");
```

Comment:
This was unused in this file. I can re-add it if it's here for some reason I'm not aware of -- to avoid DCE, perhaps?